### ISSUE TITLE
Please update for Firefox 57 Quantum

### ISSUE THREAD

**Issue by gomrcong** [kashiif/copy-urls-expert#50]
It's not working with Firefox 57, please update.
Thank you.

**Comment by kanink007** [comment]
*thumb up*
yes please, if possible, update for quantum.

did someone find a good alternative?

**Comment by nirilan** [comment]
Please update for Firefox quantum

**Comment by joshdick** [comment]
Because I needed something similar to ~this extension~ [Copy All Urls](https://chrome.google.com/webstore/detail/copy-all-urls/djdmadneanknadilpjiknlnanaolmbfk?hl=en) to work in Firefox Quantum, I just released a very similar extension: [tabclip](https://github.com/joshdick/tabclip). It feels weird to mention my own extension in Copy Urls Expert's GitHub issues, but I made tabclip for my own needs; hopefully it's useful to others as well.

Edit: Oops, I got Copy Urls Expert confused with a different extension, "Copy All Urls"! My extension tabclip is similar to "Copy All Urls", not this one. My apologies for the mixup!

**Comment by kanink007** [comment]
hi joshdick. thank you for your answer and advice.
but i cant see how tabclib is similar to copy urls expert.
imo, its not even close to it.
with copy urls expert i could mark text and copy urls.
i could even mark linked images to get the urls of them.
or i could get the single image urls if they were already full screen.

did i oversee something in tabclip?  because the only thing i can do with tabclip is: copy tab urls.

**Comment by joshdick** [comment]
@kanink007 You're totally correct! I made a silly mistake -- I confused Copy Urls Expert with yet another (Chrome-only) extension, [Copy All Urls](https://chrome.google.com/webstore/detail/copy-all-urls/djdmadneanknadilpjiknlnanaolmbfk?hl=en). I was subscribed to this GitHub issue for a different reason, and got this extension confused with Copy All Urls when I received a GitHub e-mail notification for this issue.

I've edited my previous comment accordingly. My apologies!

**Comment by AnneTheAgile** [comment]
Possibly the original author was trying to fix it here; 
https://github.com/kashiif/copy-urls-expert-web-extension/tree/webpack-setup
